platform_versioned_docs/version-24.1.0/enterprise/data-studios.mdx

```diff
@@ -90,16 +109,32 @@ This procedure describes how to configure Data Studios for Kubernetes deployment
       INITIAL_ACCESS_TOKEN: <INITIAL_ACCESS_TOKEN>
     ```
 
-1. Edit your ConfigMap named `platform-backend-cfg` for Platform and add the following environment variables:
+1. Edit the ConfigMap named `platform-backend-cfg` in the `configmap.yml` for Platform.
+    -  Add the following environment variables:
+
+       - `TOWER_DATA_STUDIO_CONNECT_URL`
+       - `TOWER_OIDC_REGISTRATION_INITIAL_ACCESS_TOKEN`: Use the same access token that you specified in the previous step
+       - `TOWER_OIDC_PEM_PATH`
```

Comment:
Value?

Reply:
@gwright99, going to shameless copy these from the environment variable YAML, but those all need to be confirmed as completed descriptions.